schedules_tools/schedule_handlers/jsonstruct.py

```diff
@@ -201,7 +188,7 @@ def export_phase_as_dict(self, phase):
         phase_export = dict()
         phase_export['name'] = phase.name
         phase_export['complete'] = phase.p_complete
-        phase_export['start'] = phase.dAcStart.strftime('%s')
-        phase_export['end'] = phase.dAcFinish.strftime('%s')
+        phase_export['start'] = phase.dStart.strftime('%s')
```

Comment:
shouldn't this whole method be removed (as phase processing is removed)?

Reply:
fixed